[2025-04-17T23:08:43.092+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parquet_to_postgres_dbt.create_tables manual__2025-04-17T23:03:39.433736+00:00 [queued]>
[2025-04-17T23:08:43.098+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parquet_to_postgres_dbt.create_tables manual__2025-04-17T23:03:39.433736+00:00 [queued]>
[2025-04-17T23:08:43.099+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-04-17T23:08:43.108+0000] {taskinstance.py:1382} INFO - Executing <Task(SQLExecuteQueryOperator): create_tables> on 2025-04-17 23:03:39.433736+00:00
[2025-04-17T23:08:43.112+0000] {standard_task_runner.py:57} INFO - Started process 245 to run task
[2025-04-17T23:08:43.115+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'parquet_to_postgres_dbt', 'create_tables', 'manual__2025-04-17T23:03:39.433736+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/parquet_to_postgres_dbt.py', '--cfg-path', '/tmp/tmp5t5_hp1o']
[2025-04-17T23:08:43.120+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask create_tables
[2025-04-17T23:08:43.170+0000] {task_command.py:416} INFO - Running <TaskInstance: parquet_to_postgres_dbt.create_tables manual__2025-04-17T23:03:39.433736+00:00 [running]> on host 8b91bf8dea38
[2025-04-17T23:08:43.247+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parquet_to_postgres_dbt' AIRFLOW_CTX_TASK_ID='create_tables' AIRFLOW_CTX_EXECUTION_DATE='2025-04-17T23:03:39.433736+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-17T23:03:39.433736+00:00'
[2025-04-17T23:08:43.249+0000] {sql.py:274} INFO - Executing: 
        CREATE TABLE IF NOT EXISTS staging_table (
            id INT,
            name TEXT,
            age INT
        );
        
        CREATE TABLE IF NOT EXISTS main_table (
            id INT PRIMARY KEY,
            name TEXT,
            age INT
        );
        
[2025-04-17T23:08:43.258+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:08:43.269+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2025-04-17T23:08:43.275+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 280, in execute
    output = hook.run(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/hooks/sql.py", line 385, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/postgres/hooks/postgres.py", line 155, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"

[2025-04-17T23:08:43.285+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=parquet_to_postgres_dbt, task_id=create_tables, execution_date=20250417T230339, start_date=20250417T230843, end_date=20250417T230843
[2025-04-17T23:08:43.296+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 9 for task create_tables (connection to server at "postgres" (172.18.0.2), port 5432 failed: FATAL:  password authentication failed for user "postgres"
; 245)
[2025-04-17T23:08:43.342+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-04-17T23:08:43.367+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
